app/src/main/java/org/p2p/wallet/striga/ui/countrypicker/delegates/StrigaCountryHeaderCellModel.kt

```diff
@@ -0,0 +1,13 @@
+package org.p2p.wallet.striga.ui.countrypicker.delegates
+
+import androidx.annotation.StringRes
+import org.p2p.uikit.model.AnyCellItem
+import org.p2p.wallet.auth.repository.Country
+
+class StrigaCountryHeaderCellModel(
+    @StringRes val titleResId: Int
+) : AnyCellItem
+
+class StrigaCountryCellModel(
+    val country: Country
+) : AnyCellItem
```

Comment:
This is wrong. 
The `CellModel` cannot be some type of the feature, it should be common. 
If you need some UI changes, then you need to update `TextViewCellModel`

Reply:
got it